nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -73,9 +73,38 @@ When NiFi first starts up, the following files and directories are created:
 
 See the <<system_properties>> section of this guide for more information about configuring NiFi repositories and configuration files.
 
+== Port Configuration
+
+=== NiFi
+The following table lists the default ports used by NiFi and the corresponding property in the _nifi.properties_ file.
+
+[options="header,footer"]
+|==================================================================================================================================================
+| Function                        |  Property                             | Default Value
+|HTTP Port                        | `nifi.web.http.port`                  | `8080`
+|HTTPS Port*                      | `nifi.web.https.port`                 | `9443`
+|Remote Input Socket Port*        | `nifi.remote.input.socket.port`       | `10443`
+|Cluster Node Protocol Port*      | `nifi.cluster.node.protocol.port`     | `11443`
+|Cluster Node Load Balancing Port | `nifi.cluster.node.load.balance.port` | `6342`
+|Web HTTP Forwarding Port         | `nifi.web.http.port.forwarding`       | blank
+|==================================================================================================================================================
+
+NOTE: The ports marked with an asterisk (*) have property values that are blank by default in _nifi.properties_.  The values shown in the table are the default values for these ports when <<tls_generation_toolkit>> is used to generate _nifi.properties_ for a secured NiFi instance.  The default Certificate Authority Port used by TLS Toolkit is `8443`.
+
+=== Embedded Zookeeper
+The following table lists the default ports used by an <<embedded_zookeeper>> and the corresponding property in the _zookeeper.properties_ file.
+
+[options="header,footer"]
+|==================================================================================================================================================
+| Function                                         | Property       | Default Value
+|Zookeeper Client Port                             | `clientPort`   | `2181`
+|Zookeeper Server Quorum and Leader Election Ports | `server.1`     | `localhost:2888:3888`
```

Comment:
In zookeeper.properties, the default values are commented:
```properties
# server.1=nifi-node1-hostname:2888:3888
# server.2=nifi-node2-hostname:2888:3888
# server.3=nifi-node3-hostname:2888:3888
```
Technically this means that the default values for `server.1` to `server.N` are empty.  The note below references that examples are commented out, which is good, but I think the port list should indicate that the `server.N` properties are empty by default.

Reply:
This looks good, with `blank` being changed to _`none`_.

Reply:
Thanks, will make this change also.